core/Metrics.php

```diff
@@ -431,6 +436,7 @@ public static function getDefaultMetricTranslations()
             'nb_conversions'                => 'Goals_ColumnConversions',
             'revenue'                       => 'General_ColumnRevenue',
             'nb_hits'                       => 'General_ColumnPageviews',
+            'hits'                          => 'General_ColumnHits',
```

Comment:
Oh no... this might create a lot of confusion.. All hits (as in all events, views, actions) vs just page views.. 😕

Reply:
Should we/could we maybe use `all_hits` everywhere? Also as noted in the other comment as it doesn't have the `nb` prefix, is that just an inconsistency we acquired over time that some numeric metrics don't have the prefix and some do?

Reply:
Or possibly, as the description suggests, `interactions` or `all_interactions`?

Reply:
The "problem" is, that we already had `hits` defined. It wasn't used in core yet, but custom reports is already using that metrics:
https://github.com/matomo-org/matomo/blob/9a3ef94df61682e7d543f3f8b438c368f6eb71a6/plugins/CoreHome/Columns/LinkVisitActionId.php#L20-L33

I was considering to rename the metric, but that would break custom reports. Reusing `nb_hits` would in theory also be possible, but that has the problem, that the default metric description and name would no longer work, as it would need to differ between the report type. And using a completely different name would end up in having the metric twice in custom reports.
The naming of our metrics is bad in a couple of cases, but it's hard to get that right again without fully breaking BC.

Reply:
Thanks for the clarification.

Note for future ourselves - we also discussed that in other team meetings and during a subsequent round of peer reviewing and decided to leave this as is.